staging/src/k8s.io/client-go/transport/config.go

```diff
@@ -80,6 +84,11 @@ func (c *Config) HasCertAuth() bool {
 	return len(c.TLS.CertData) != 0 || len(c.TLS.CertFile) != 0
 }
 
+// IsDebugConcurrent returns whether we should be debugging concurrent state.
```

Comment:
Tempted to suggest we head more toward accessors and better obfuscation. However as most of the Config is already public not sure its worth worrying about.